openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -154,6 +154,16 @@ jQuery(function () {
             .then(module => module.initAddCoverImport());
     }
 
+    if (document.getElementsByClassName('manageCovers').length) {
+        import(/* webpackChunkName: "image_change" */ './image_change')
+            .then(module => module.initChangeImageImport());
+    }
+
+    if (document.getElementsByClassName('imageSaved').length) {
```

Comment:
Can we merge these into a single import? Using asynchronous `import` means the JS is split into 2 bundles. I'd like it split into 1 bundle as this is one feature. The init function for that imported bundle can do the checks for imageSaved and manageCovers

Reply:
After a few days of testing and trying to figure this one out, I haven't been able to make it work. For whatever reason, it appears the addition of iframes to the DOM are not registered for both the Document and JQueryStatic objects inside this file. The only way I could make jQuery or regular JavaScript find the elements needed to be operated on inside the iframes was to call the JavaScript for the iframes inside of index.js. Let me know if you have any other suggestions though. I just couldn't seem to make the above work, nor any other solutions I tried looking for on the internet. Sorry!

Reply:
This is fine. We're using the same module, so at least we should be utilizing caching. I can take a look at this separately but doesn't need to block.